pyro_risks/datasets/model_dataset.py

```diff
@@ -0,0 +1,79 @@
+from pyro_risks.datasets import NASAFIRMS_VIIRS, ERA5Land
+from pyro_risks.datasets.utils import get_intersection_range
+from pyro_risks.datasets.fwi import GwisFwi
+
+import pandas as pd
+
+__all__ = ["ModelDF"]
+
+
+class ModelDF(pd.DataFrame):
```

Comment:
Perhaps we can find a more informative class name (and module name) :)

Reply:
alright, I'm open to suggestions

Reply:
Perhaps something along these lines: 
- prepData ? 
- modelingData ?

Reply:
`WildfireTimeSeries` would be a bit general, but it's a start?
Or what about "ERAVIIRS"?

Reply:
done with `MergedEraFwiViirs`, I hope you like it ;-)